docs/script.rst

```diff
@@ -215,6 +215,40 @@ In the preceding example, ``blastp`` and its ``-in``, ``-out``, ``-db`` and ``-h
 their arguments are effectively a single line.
 
 
+Functions
+---------
+
+Functions can be defined using the following syntax::
```

Comment:
Is it worth pointing out that functions are a groovy thing and not Nextflow, with a link to some groovy docs?